packages/language-support/src/tests/syntaxValidation/syntacticValidation.test.ts

```diff
@@ -820,14 +848,14 @@ describe('Syntactic validation spec', () => {
       }),
     ).toEqual([
       {
-        message: "Expected any of '=', ')', '{', ':', IS, WHERE or a parameter",
+        message: `Invalid input '"foo"': expected a graph pattern, a parameter, ')', ':', 'IS', 'WHERE' or '{'`,
         offsets: {
-          end: 14,
+          end: 15,
           start: 9,
         },
         range: {
           end: {
-            character: 14,
+            character: 15,
```

Comment:
Because of the way we are computing the positions finish in the mono repo, some errors are spanning new characters now.

That is, when we receive a syntax error, we just know where the token starts, but not where it finishes. And the only approach we have in that case is to consider the token finishes at the next non space position.